velox/docs/monthly-updates/march-2023.rst

```diff
@@ -0,0 +1,75 @@
+********************
+March 2023 Update
+********************
+
+Documentation
+=============
+
+ * Add :doc:`Chapter 1  of the Programming Guide <../programming-guide/chapter01>`.
+ * Add blog post on `continuous integration and packaging. <https://velox-lib.io/blog/velox-build-experience>`_
+
+
+Core Library
+============
+
+ * Add `printVector` helper function.
+ * Add support for `copy_from` in GenericWriter.
+ * Add preliminary support for OffProcessExpressionEval operator.
+ * Add quota enforcement checks when allocating large chunks of memory.
+ * Deprecate `reallocate` Api from memory allocator.
+ * Optimize null aware anit-join when used with filter.
+ * Remove usage of CppToType.
+
+
+Presto Functions
+================
+
+ * Add :func:`from_utf8`, :func:`hmac_md5` functions.
+ * Add :func:`any_match`, :func:`all_match` and :func:`none_match` functions.
+ * Add support for DECIMAL types in :func:`min` and :func:`max` aggregate functions.
+
+
+Spark Functions
+===============
+
+ * Add :spark:func:`trim`, :spark:func:`ltrim`, :spark:func:`rtrim`  function.
+ * Add :spark:func:`sha2` function.
+
+
+Hive Connector
+==============
+
+ * Add support for multi-level sub field pruning.
+ * Add support for estimating parquet page headers using `ThriftStreamingTransport`.
+ * Fix crash when reading lazy vectors.
```

Comment:
We can remove this, as it's just a bug fix and nothing user should note on their side